package.json

```diff
@@ -299,7 +299,8 @@
 		"prewhybundled": "cross-env-shell CALYPSO_ENV=production EMIT_STATS=withreasons NODE_ARGS=--max_old_space_size=8192 npm run -s build-client",
 		"whybundled": "whybundled stats.json",
 		"composite-checkout-demo": "webpack-dev-server --config ./packages/composite-checkout/webpack.config.demo.js --mode development",
-		"components:storybook:start": "start-storybook -c packages/components/.storybook"
+		"components:storybook:start": "start-storybook -c packages/components/.storybook",
+		"components:storybook:build": "build-storybook -c packages/components/.storybook -o packages/components/storybook-static"
```

Comment:
I would be nice if all the assets the HTTP server needs at runtime were concentrated in a small number of directories (`public/`, `build/`, ...) instead of being spread all over the source tree.

Can the Storybook build output directory be changed to `public/storybook`?

Reply:
Agreed. Splitting the old mixed `public` into a source-only `static` and a built `public` made things a lot easier in the past (config files, npm scripts, etc.), so I think @jsnajdr's suggestion would bring similar benefits here.